src/components/blinds/blinds.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import "./style.css";
+import blindClosed from "./blinds-closed.svg";
+import blindOpen from "./blinds-open.svg";
+
+const imgage = [blindClosed, blindOpen];
```

Comment:
tohle tu nic nedělá, že? to pole obrázků jsme předtím ve cvičeních používali ve chvíli kdy jsme chtěli na obrázky mapovat třeba čísla 0-6 (kostka). A bylo to kratší než dělat několikrát if/elseif/elseif/else... tady pro on/off stačí ty obrázky referencovat přímo, jak to děláš 👍